content/docs/atom-guide/look-dev/_index.md

```diff
@@ -14,4 +14,4 @@ This section contains the information about creative tools and concepts for look
 | [Textures](textures/) | Learn about how textures are supported and processed in Atom. |
 | [Shading](shaders/) | Learn about shading in Atom. |
 | [Color Management](color-management/) | Learn about the color space conversion workflow used in Atom. |
-| [Material Types and Shaders](get-started-materialtypes-and-shaders/) | A beginner-level tutorial introducing material types and shaders in Atom. Learn how to create a custom material type with a simple AZSL shader. | 
\ No newline at end of file
+| [Tutorials](tutorials) | Tutorials to get started on creating custom additions and changes. |
```

Comment:
I have some suggestions for structuring this table of contents. 

First, I highly suggest removing the **Look Development > Tutorials** section, since it just links to the rendering tutorials in the Learning Guide. You can do provide that link here instead. 

Then, update the following line
```suggestion
| [Tutorials](/docs/learning-guide/tutorials/rendering/)) | A collection of tutorials about rendering and look development in O3DE, located in the Learning Guide. |
```